lib/suggestion-list.js

```diff
@@ -227,13 +236,12 @@ export default class SuggestionList {
   }
 
   showAtBeginningOfPrefix (editor, prefix, followRawPrefix = false) {
-    if (!editor) {
```

Comment:
Isn't an early return with no nesting of meaningful code preferable to the alternative in this function?